.github/workflows/post-merge.yaml

```diff
@@ -124,7 +124,10 @@ jobs:
           go-version: ${{ steps.go_version.outputs.go_version }}
 
       - name: Build Darwin
-        run: make ci-build-darwin ci-build-darwin-arm64-static
+        run: |
+          make ci-build-darwin GOARCH=amd64
+          make ci-build-darwin GOARCH=arm64
```

Comment:
We can skip this in-line with previous releases and build only static binary for arm64.

Reply:
Also we need to update the `pull-request` and `post-tag` workflows.

Reply:
Thank you for your review.
Fixed.
https://github.com/open-policy-agent/opa/pull/6721/commits/c5654db137faef2b78d121f684bbad2ea61e5db2
https://github.com/open-policy-agent/opa/pull/6721/commits/f76246f735e4a9ed2db0b3b7f9341554016e65ab